src/scripts/achievements/AchievementHandler.ts

```diff
@@ -137,7 +137,7 @@ class AchievementHandler {
         AchievementHandler.addAchievement('Keeping Oak Really Busy', 'Capture 75 unique Pokémons', new CaughtPokemonRequirement(75), 0.15);
         AchievementHandler.addAchievement('Surpassing Ash', 'Capture 88 unique Pokémons', new CaughtPokemonRequirement(88), 0.05);
         AchievementHandler.addAchievement('I Wanna be The Very Best', 'Capture 100 unique Pokémons', new CaughtPokemonRequirement(100), 0.20);
-        AchievementHandler.addAchievement('I Should Probably Take a Break', 'Complete the Kanto Pokédex!', new CaughtPokemonRequirement(151), 0.50);
+        AchievementHandler.addAchievement('I Should Probably Take a Break', 'Complete the Kanto Pokédex!', new CaughtUniquePokemonsByRegionRequirement(GameConstants.Region.kanto), 0.50);
```

Comment:
PR looks good to me,
Do you think we should make this bonus apply the Kanto region percentage instead of the global percentage?

_not really sure which makes more sense_

Reply:
i dont know about that, i dont know how the bonus really works, I have in my save more than 100%, its not global?
Dont make much sense, for me, just for Kanto cause when you complete Kanto you travel to other regions, just come back for quests or shinies.